### DISCUSSION TITLE
ERROR: exit code 2

### DISCUSSION THREAD

**Discussion by dizzyfizzy101** [qnblackcat/uYouPlus#1465] [Q&A]
### I try to create my IPA exactly as it stands in the guide, but everytime i run the workflow i get this error code:
![image](https://github.com/qnblackcat/uYouPlus/assets/170124129/d7c88000-096a-45d9-bd84-4948379322c4)
_what do i do?_
**i've already tried multiple things to fix this such as:**
- **trying** another direct link to a different decrypted **IPA** file
- **changing** the Workflow file from 
    > runs-on: macos-12 => runs-on: macos-11

**please help.**
thanks

**Comment by matheus-nicacio** [comment]
I have the same problem. I've tried to solve it in several ways and I couldn't :(
